### ISSUE TITLE
Unable to set selector for fan oscillate / selector unable to get state from attributes

### ISSUE THREAD

**Issue by SodaWithoutSparkles** [Nerwyn/service-call-tile-feature#39]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
4.0.1
**Last working release (if known):**

**Browser and Operating System:**
Chrome 127, android app

**Description of problem:**

Unable to set a selector highlight state based on fan's oscillating attribute. 

**Javascript errors shown in the web inspector (if applicable):**

```
N/A
```

**Your Full Remote Config**

This is a basic reproducable config.

```yaml

type: tile
entity: fan.fan1
show_entity_picture: false
vertical: false
hide_state: false
state_content: percentage
features:
  - type: custom:service-call
    entries:
      - type: selector
        entity_id: fan.fan1
        options:
          - type: button
            entity_id: fan.fan1
            option: 'true'
            tap_action:
              action: call-service
              service: fan.oscillate
              target:
                entity_id:
                  - fan.fan1
              data:
                oscillating: true
            value_attribute: oscillating
            icon: mdi:arrow-oscillating
          - type: button
            entity_id: fan.fan1
            option: 'false'
            tap_action:
              action: call-service
              service: fan.oscillate
              target:
                entity_id:
                  - fan.fan1
              data:
                oscillating: false
            value_attribute: oscillating
            icon: mdi:arrow-oscillating-off
        value_attribute: oscillating

```

**Additional information:**
It cannot highlight the relevant option automatically. When I press the selector. the fan did respond and the selector highlight did update. However, when I set the fan to not oscillate in any other way, it did not update the highlight.

I suspect this issue is due to the card not knowing the current fan oscillating state.

**Comment by SodaWithoutSparkles** [comment]
This is the fan's states:

```yaml
preset_modes:
  - Normal
  - Nature
direction: null
oscillating: false
percentage: 99
percentage_step: 1
preset_mode: Normal
icon: mdi:fan
friendly_name: Fan
supported_features: 15
```

**Comment by Nerwyn** [comment]
It could be because the option is the Boolean as a string while the actual value is a Boolean. Can you try changing the options to Booleans using the YAML editor? I think this is the same bug as #38 (coincidentally posted a few hours before you).

**Comment by Nerwyn** [comment]
@SodaWithoutSparkles  if you could update your config and try the 4.0.2 alpha, it should fix non-string selector value option comparisons.

**Comment by SodaWithoutSparkles** [comment]
Can confirm the string casting works. I think this is ready for prod, but lets wait until the other issue.

**Comment by Nerwyn** [comment]
Fixed in [4.0.2](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/4.0.2).
